backend/apps/medical_catalog/services/hospital_item_price_service.py

```
@@ -12,23 +64,52 @@ def create(*, data: dict, user=None) -> HospitalItemPrice:
+            if _is_unique_constraint_violation(e):
+                raise DuplicateError("HospitalItemPrice", message="HospitalItemPrice with these values already exists") from e
+            else:
+                # Other integrity errors (NOT NULL, FK, etc.) - raise InvalidValueError
+                raise InvalidValueError(
+                    field=None,
+                    message="Database constraint violation",
+                    details={"error": str(e)}
+                ) from e
 
     @staticmethod
     def update(*, price_id: str, data: dict, user=None) -> HospitalItemPrice:
```

Ruff (0.14.2)

Unused static method argument: `user`

(ARG004)